tests/test_all_rules.py

```diff
@@ -370,6 +415,16 @@ def test_rule_file_name_tactic(self):
             rule_err_str = '\n'.join(bad_name_rules)
             self.fail(f'{error_msg}:\n{rule_err_str}')
 
+    def test_bbr_in_correct_dir(self):
+        """Ensure that BBR are in the correct directory."""
+        for rule in self.bbr:
+            self.assertEqual(rule.path.parent.name, 'rules_building_block',
+                             f'{self.rule_str(rule)} should be in the rules_building_block directory')
+
+        for rule in self.all_rules:
+            if rule.path.parent.name == 'rules_building_block':
+                self.assertIn(rule, self.bbr, f'{self.rule_str(rule)} should be in the rules_building_block directory')
```

Comment:
Is this another test under one unittest? May want to break it out (if it makes sense). Also, what is it testing?

Reply:
The inverse test of the first. First one checks all bbr rules to see if they are in `rules_building_block`. This one checks all of the non-bbr to see if they are in the `rules_building_block` directory and errors if any of them are.

Reply:
Does that mean rules in self.all_rules does NOT include bbr rules?

Reply:
Correct, it does not in the current implementation.